packages/easy-email-extensions/src/components/Form/RichTextField/index.tsx

```diff
@@ -116,10 +118,10 @@ function FieldWrapper(
 
   return (
     <>
-      {contentEditableType === ContentEditableType.RichText && (
+      {contentEditableType === ContentEditableType.RichText && richToolBarVisible && (
         <RichTextToolBar onChange={debounceCallbackChange} />
       )}
-      <InlineText {...rest} onChange={debounceCallbackChange} />;
+      <InlineText {...rest} onChange={debounceCallbackChange} />
     </>
   );
```

Comment:
Sorry, it's base on shadow dom. But you can update it.

It looks like you just need `InlineTextField`, check this
<img width="702" alt="image" src="https://user-images.githubusercontent.com/20615214/230128626-05bb74c3-2619-4bf5-bc9a-e7db343ca1b5.png">.

Reply:
Changing my component from 

```
const NoLayoutEasyEmailEditor = () => {
  const { height: containerHeight } = useEditorProps();
  const { focusIdx } = useFocusIdx();
  return (
    <div style={{ height: containerHeight }}>
      <ShadowDom>
        <RichTextField idx={focusIdx} richToolBarVisible={false} />
      </ShadowDom>
      <EditEmailPreview />
    </div>
  );
};

export default NoLayoutEasyEmailEditor;
```

to this

```
const NoLayoutEasyEmailEditor = () => {
  const { height: containerHeight } = useEditorProps();
  const { focusIdx } = useFocusIdx();
  return (
    <div style={{ height: containerHeight }}>
      <ShadowDom>
        <InlineTextField idx={focusIdx} name="NoLayout" />
      </ShadowDom>
      <EditEmailPreview />
    </div>
  );
};

export default NoLayoutEasyEmailEditor;
```

is making `EmailEditorProvider` function re-render but it doesn't change the actual form values of the input I'm modifying. 

@m-Ryan  what am I missing ? 🤔  I feel there is maybe a bit of magic I'm not aware of 😛

Reply:
It seems I must add a `name` property on that component, something I didn't have to do with the `RichTextField`. If I apply one by default it updates that property name. In my use-case, I can't afford to name anything. I essentially only render this component so I get the benefit of having the form update at the right place each time something is edited in any of the inputs

Reply:
I was able to have a decent solution with the RichTextToolBar appearing... 

It doesn't change the fact that nobody will ever be able to use part of the editor without the richTextToolBar if they need but for now, I don't need it :P 

I updated my PR to expose providers instead so I can have all the features available from the RichTextToolBar working.

```
const NoLayoutEasyEmailEditor = () => {
  const { height: containerHeight } = useEditorProps();
  const { focusIdx } = useFocusIdx();
  return (
    <div style={{ height: containerHeight }}>
      <SelectionRangeProvider>
        <PresetColorsProvider>
          <ShadowDom>
            <RichTextField idx={focusIdx} />
          </ShadowDom>
          <EditEmailPreview />
        </PresetColorsProvider>
      </SelectionRangeProvider>
    </div>
  );
};
```